combat-trainer.lic

```diff
@@ -384,6 +392,12 @@ class LootProcess
     return if Time.now - @loot_timer < @loot_delay
 
     game_state.mob_died = true
+    if @last_rites_toggle
```

Comment:
Why not use the conditional from line 257 here, so you don't need the separate `@last_rites_toggle` boolean?